packages/bpk-docs/src/components/neo/PageHead/PageHead.js

```diff
@@ -51,8 +51,8 @@ type Props = {
 const PageHead = (props: Props) => (
```

Comment:
I wonder if there is now a better name for this, as it's not so much the page but the block of information. Can't think of one myself, though 🤔